Python/coreconfig.c

```diff
@@ -69,6 +69,9 @@ _Py_wstrlist_copy(int len, wchar_t **list)
     assert((len > 0 && list != NULL) || len == 0);
     size_t size = len * sizeof(list[0]);
     wchar_t **list_copy = PyMem_RawMalloc(size);
+    if (list_copy == NULL) {
+        return NULL;
+    }
```

Comment:
There is another bug below: _Py_wstrlist_clear(i, list), it should be list_copy, not list.

Reply:
Fixed, thanks!